src/_content/posts/keynote-carson.md

```diff
@@ -0,0 +1,69 @@
+---
+author:  Communications Team
+category: General
+published_datetime: 2025-07-27 06:00:00-04:00 
+title: "Announcing Our Keynotes: Carson Gross"
+
+cover:
+    url: /assets/img/2025/Carson.png
+    alt: "Announcing Our Keynotes"
+---
+
+### We're honored to announce our first keynote speaker for DjangoCon US 2025: Carson Gross
+
+We had the opportunity to sit down with Carson and ask him a few questions about his work, his upcoming talk, and what he's looking forward to at DjangoCon US 2025.
+
+## How do you use Django?
+
+> I do not use Django very often because my main profession is as a teacher and we mainly use Java here. However, the Django community has embraced HTMX, a web library that I created, and so I have a lot of friends in the community.
+
+## Can you give us a teaser about what your talk is about?
+
+> I will be speaking about the philosophy of Nintendo Game Designer Gunpei Yokoi who created the original Game Boy.  His philosophy translates as "Lateral Thinking with Withered (or Seasoned) Technology" and I am going to apply that thinking to web development, particularly the combination of HTMX & Django.
+
+## What are you most looking forward to at DjangoCon US 2025?
+
+> I am looking forward to meeting people and talking about hypermedia.
+
+## What are your favourite topics at conferences?
+
+> I enjoy technical deep dives in particular, especially when they start from first (or simple) principles.
+
+## How do you view the connections made at conferences?
+
+> I love making connections, but I am pretty remote living in Montana so they tend to be relatively short lived unfortunately.
+
+## Can you tell us about your career journey?
```

Comment:
Consider moving the question from line 36 to the first question for better flow.